### PR TITLE
New stuff

### PR DESCRIPTION
**Pull request by jackall3n**

### CODE REVIEW

File: BlueCheese/Hubs/LobbyHub.cs [hunk 1]
```diff
@@ -18,12 +18,12 @@ public LobbyHub(IGameManager gameManager, ILogger<LobbyHub> logger)
...
         }
 
-        public async Task ClientStartedNewGame(NewGameStarted newGame)
+        public async Task<IGame> ClientStartedNewGame(NewGameStarted newGame)
         {
             if(newGame==null) throw new ArgumentNullException(nameof(newGame));
             _logger.LogInformation("LobbyHub.ClientStartedNewGame {@newGame}", newGame);
 
-            await _gameManager.StartNewGameAsync(Context.ConnectionId, newGame).ConfigureAwait(false);
+            return await _gameManager.StartNewGameAsync(Context.ConnectionId, newGame).ConfigureAwait(false);
```
Comment (chrisw000) [inline BlueCheese/Hubs/LobbyHub.cs]: the docs show these client initiated methods as returning nothing... separate signals sent when things happen....

does this actually return the game object? and if so - can you update the lobby.js in this project to print it out to console or something like that?

**Review by chrisw000 — COMMENTED** [review]
see comments

File: BlueCheese/HostedServices/Bingo/Game.cs [hunk 2]
```diff
@@ -80,8 +83,10 @@ public async Task SpawnAsync(string connectionId, NewGameStarted newGameStarting
                 ConnectionId = connectionId,
                 GameId = GameId
             };
+            
             await AddPlayerAsync(joinGame).ConfigureAwait(false);
             await _lobbyHubContext.Clients.All.LobbyNewGameHasStarted(this).ConfigureAwait(false);
+            await _lobbyHubContext.Clients.Client(connectionId).LobbyGameCreatedSuccess(this, $"Game with id {GameId} created successfully").ConfigureAwait(false);
```
[the remark this answers is not in the document]
Comment (jackall3n) [reply BlueCheese/HostedServices/Bingo/Game.cs]: yeah actually this isnt needed anymore, i ended up just returning the object with Guid in the original call

File: BlueCheese/Hubs/LobbyHub.cs (hunk 1 above)
Comment (jackall3n) [reply BlueCheese/Hubs/LobbyHub.cs]: yeah, i'll add that in

Comment (jackall3n) [reply BlueCheese/Hubs/LobbyHub.cs]: what docs you referring to?

Comment (jackall3n) [reply BlueCheese/Hubs/LobbyHub.cs]: Game object is used for the front end to be able to go to the game using the game GUID.

**Review by chrisw000 — APPROVED** [review]

**Pull request merged**